src/ImageDeleteModal.jsx

```diff
@@ -81,11 +75,20 @@ export const ImageDeleteModal = ({ imageWillDelete, onAddNotification }) => {
         }
     };
 
+    const imageName = repoTags[0]?.split(":")[0].split("/").at(-1) ?? "";
```

Comment:
This is about intermediate images I assume. So:
![image](https://github.com/cockpit-project/cockpit-podman/assets/67428/79db44cf-d460-4b90-8537-ae20be46e924)

Versus:

![image](https://github.com/cockpit-project/cockpit-podman/assets/67428/d3e1c1ce-5d38-42b6-9a18-eaa91a66782f)

So nothing breaks, so that's fine!